library/summary.ml

```diff
@@ -107,7 +107,7 @@ let unfreeze_summaries fs =
     try fold id decl state
     with e when CErrors.noncritical e ->
       let e = CErrors.push e in
-      Feedback.msg_error
+      Feedback.msg_warning
```

Comment:
I don't know the implications of this. Are warning and error feedbacks interpreted in the same way, say in CoqIDE? It looks strange to print a warning message that is actually an error, right? Should it be fixed in another way?

Reply:
This error should be pretty uncommon. I wonder why it's not an anomaly actually.

Reply:
So it is really a bit of a remaining mess, because what we want to do in an error is raise an exception and let the handler print it. But some handlers want to "decorate" the printing exception. So I dunno how to proceed.

Reply:
This PR just fixed the msg_error not being printed, even in a non-optimal way in these two [extremely rare] cases.